schema/transportation/segment.yaml

```diff
@@ -455,23 +526,23 @@ properties:
           properties: { is_at_most: { allOf: [ { "$ref": "#/$defs/propertyDefinitions/weightValueWithUnit" } ] } }
         - required: [is_less_than]
           properties: { is_less_than: { allOf: [ { "$ref": "#/$defs/propertyDefinitions/weightValueWithUnit" } ] } }
-    sequenceEntry:
+    subSegmentReference:
       description: >-
-        A segment/connector pair in a prohibited transition sequence.
+        A segment/connector pair used to reference a part of a segment 
+        that can have connections to other segments. 
+        Used in describing prohibited transition sequences and source 
+        segments for destinations.
       type: object
       required: [connector_id, segment_id]
       properties:
         connector_id:
           description: >-
-            Identifies the point of physical connection between the
-            previous segment in the sequence and the segment in this
-            sequence entry.
+            Identifies the point of physical connection between this segment 
+            and other segments.
```

Comment:
I don't understand this description change. It converts from something well-defined/rigorous into something vague. Why would we want to do that?

---

If I'm understanding right, I think this is motivated by a difference between prohibited transitions and destinations, where in PT the sequence comes *after* the current segment and in D it comes *before* the current segment.

In the circumstances, it's probably better to duplicate the sequence concept into two types, `followingSequence` for PT and ``precedingSequence` for D, so that we can still have rigorous definitions for both.

Reply:
reverted this change